### PR TITLE
katib: Update LLM HP tuning guide to clarify tunable fields and fix resource section

### PR DESCRIPTION
**Pull request by SanthoshToorpu**
…es config, remove unsupported params

**Checklist:**
- [x] You have [signed off your commits](https://www.kubeflow.org/docs/about/contributing/#sign-off-your-commits)
- [x] Ensure you follow best practices from our guide. [Contributing](https://github.com/kubeflow/website/blob/master/content/en/docs/about/contributing.md). 
- [x] You have included screenshots when changing the website style or adding a new page.

**Description of your changes:**
As per andrey's recommendation I added the changes mentioned in the following screenshots attached. A minimal work.
image (2)
image (1)
image

### Issue

Closes: #2522

### Labels

/area katib 

 /area website 

---

### CODE REVIEW

**Comment by SanthoshToorpu** [comment]
@helenxie-bit please look into the changes

**Review by Arhell — COMMENTED** [review]
/ok-to-test

**Comment by SanthoshToorpu** [comment]
@andreyvelich as per @helenxie-bit instructions and after reading the thread i fixed 2/3 issues except for the params you told to remove irrelevant params. I downloaded the docker image rn but can you be more specific on what to be removed and what not?

File: content/en/docs/components/katib/user-guides/llm-hp-optimization.md [hunk 1]
```diff
@@ -165,6 +109,7 @@ In addition to Hugging Face, you can integrate with S3-compatible object storage
 from kubeflow.storage_initializer.s3 import S3DatasetParams
 ```
 
+
 #### S3DatasetParams
```
Comment (helenxie-bit) [inline content/en/docs/components/katib/user-guides/llm-hp-optimization.md:113]: I suggest we also move this part to the Training Operator doc and cross-reference it from this doc: https://github.com/kubeflow/website/blob/master/content/en/docs/components/trainer/legacy-v1/user-guides/fine-tuning.md

**Review by helenxie-bit — COMMENTED** [review]
Thank you for this contribution! It helps a lot to make the user guide clearer. Here are a few suggestions:

**Comment by helenxie-bit** [comment]
Also for this part: https://www.kubeflow.org/docs/components/katib/user-guides/llm-hp-optimization/#key-parameters-for-llm-hyperparameter-tuning, can you remove parameters `objective`, `base_image`, and `parameters`? Since they will not be used when optimizing hyperparameters for LLMs.

**Comment by SanthoshToorpu** [comment]
> > @andreyvelich as per @helenxie-bit instructions and after reading the thread i fixed 2/3 issues except for the params you told to remove irrelevant params.
> > image
> > I downloaded the docker image rn but can you be more specific on what to be removed and what not?
> 
> I think we need to remove unused parameters in this part: https://www.kubeflow.org/docs/components/katib/user-guides/llm-hp-optimization/#key-parameters-for-llm-hyperparameter-tuning. Can you remove parameters objective, base_image, and parameters? Since they will not be used when optimizing hyperparameters for LLMs.

Okay so only those threee params?

File: content/en/docs/components/katib/user-guides/llm-hp-optimization.md (hunk 1 above)
Comment (SanthoshToorpu) [reply content/en/docs/components/katib/user-guides/llm-hp-optimization.md:113]: The S3 data params part? I guess that has rather less relevance to training operator. However if you are referencing to the example I believe that we should have a lil snippet that gives a high level overview as most people are lazy enough to  browse nested links....

I'll commit with the three params removed. But please give me a heads up wrt the above suggestion

Comment (helenxie-bit) [reply content/en/docs/components/katib/user-guides/llm-hp-optimization.md:113]: Yes, `S3DatasetParams` is another way to define an external dataset, just like `HuggingfaceDatasetParams`. So I think it makes sense to move its explanation into the Training Operator documentation as well.

**Comment by helenxie-bit** [comment]
> > > @andreyvelich as per @helenxie-bit instructions and after reading the thread i fixed 2/3 issues except for the params you told to remove irrelevant params.
> > > image
> > > I downloaded the docker image rn but can you be more specific on what to be removed and what not?
> > 
> > 
> > I think we need to remove unused parameters in this part: https://www.kubeflow.org/docs/components/katib/user-guides/llm-hp-optimization/#key-parameters-for-llm-hyperparameter-tuning. Can you remove parameters objective, base_image, and parameters? Since they will not be used when optimizing hyperparameters for LLMs.
> 
> Okay so only those threee params?

Yes, all other parameters may be used when optimizing hyperparameters for LLMs.

**Comment by SanthoshToorpu** [comment]
@helenxie-bit :Great! Maybe here it's better to use a different title since we include S3DatasetParams in this part too:

how about
## Dataset and Model Parameter Classes
in legacy trainer docs

**Comment by helenxie-bit** [comment]
> @helenxie-bit :Great! Maybe here it's better to use a different title since we include S3DatasetParams in this part too:
> 
> how about
> 
> ## Dataset and Model Parameter Classes
> in legacy trainer docs

That sounds good to me.

**Comment by SanthoshToorpu** [comment]
Please review

**Comment by helenxie-bit** [comment]
Thanks for the contribution! LGTM! Please have a review when you have time @andreyvelich @mahdikhashan

**Comment by juliusvonkohout** [comment]
@andreyvelich @Electronic-Waste  is this ready to be merged ?

**Comment by mahdikhashan** [comment]
> @andreyvelich @Electronic-Waste is this ready to be merged ?

I think It was pending for my review, I had a quick look and seems good to me to be merged. as i remember it to be a link to old doc and further improvment.

**Comment by juliusvonkohout** [comment]
/approve
then
